test/puppetlabs/puppetdb/command_test.clj

```diff
@@ -1279,11 +1322,13 @@
              {:certname "bar.example.com"
               :command {:command (command-names :deactivate-node)
                         :version 2
-                        :payload "bar.example.com"}}
+                        :payload (json/generate-string "bar.example.com")}}
              {:certname "bar.example.com"
               :command {:command (command-names :deactivate-node)
                         :version 1
-                        :payload (json/generate-string "bar.example.com")}}]]
+                        :payload (-> "bar.example.com"
+                                     json/generate-string
+                                     json/generate-string)}}]]
```

Comment:
Yeah. When an "old format" deactivate node command comes in, the certname **is** the payload. In http/command, we parse that payload, then generate-string it (that code is [here](https://github.com/puppetlabs/puppetdb/blob/master/src/puppetlabs/puppetdb/http/command.clj#L150)). In version 1 of deactivate node, our command processing code (not http/command, but puppetdb/command) expects a json escaped string. The outer "generate-string" gets unwrapped by http/command.clj and for version 1, puppetdb/command.clj is expecting json escaped there as well.